Header header

float32 acc_x
float32 acc_y
float32 acc_z

float32 gyr_x
float32 gyr_y
float32 gyr_z

int16 joy_front
int16 joy_side

int16 battery_power
float32 battery_current

float32 speed

bool power_on
int16 error
int16 id_position
